tests/functional/basic_usage.js

```diff
@@ -4,7 +4,7 @@ const assert = require('assert');
 const PassThrough = require('stream').PassThrough;
 const pass = new PassThrough;
 
-const Logger = require('werelogs').Logger;
+const Logger = require('werelogs').Werelogs;
```

Comment:
If you want to test backwards compatibility with my original fix, you should leave this line unchanged -> That would allow confirming that most projects do not require changing and that the backwards-compat works as expected.